app/components/placement/summary_component.html.erb

```diff
@@ -1,5 +1,5 @@
 <li class="app-search-results__item">
-  <div class="govuk-grid-column-three-quarters govuk-!-padding-left-0">
+  <div class="govuk-grid-column-full govuk-!-padding-left-0">
     <h2 class="app-search-result__item-title">
       <%= govuk_link_to(placements_placement_path(placement), no_visited_state: true) do %>
         <span><%= school.name %></span>
```

Comment:
I don't think we need this wrapper `<div>` – the grid class and padding override isn't helpful here.

Could we pull the `<h2>` up so it's a direct descendent of the `<li>` please?

```suggestion
  <h2 class="app-search-result__item-title">
    <%= govuk_link_to(placements_placement_path(placement), no_visited_state: true) do %>
      <span><%= school.name %></span>
```

Reply:
Resolved in [3857392](https://github.com/DFE-Digital/itt-mentor-services/pull/689/commits/3857392603bb84909be6d41dccc73a3fcfe0c35a)